tests/api/test_security.py

```diff
@@ -0,0 +1,81 @@
+import requests
+import unittest
```

Comment:
This import is actually useless as t's never used, unless I am missing something?